docs/guides/node-installation.md

```diff
@@ -27,23 +36,24 @@ sidebar_label: Installing Nodes
 </tr>
 </table>
 
-See our [GitHub repository](https://github.com/lacchain/eosio-network) for more information about nodes in LACChain EOSIO.
+See our [repository on GitHub](https://github.com/lacchain/eosio-network) for more information about nodes in LACChain EOSIO. If you want to install and configure nodes from the command line, consult the [README.md](https://github.com/lacchain/eosio-network/blob/master/README.md) of our repository on GitHub, before then make sure you have installed the [development environment](./development-environment) for EOSIO.
+
```

Comment:
include a link to the following guides https://guide.eoscostarica.io/docs/tutorials/node-installation

perhaps some more content about actual steps to install configure and  run nodeos would be helpful